docs/dm/settings.md

```diff
@@ -24,62 +36,88 @@ Remove Session **PERMANENTLY** removes the campaign from the PlanarAlly server.
+See the [lighting docs](/docs/dm/light-shadows/) for more information on this topic.
 
 ### FoW Opacity
 
-This simple setting allows you to choose what the opacity of the hidden areas is just for you. Players will always have this setting as 1.0 for themselves and can't edit this.
+This setting allows the DM to choose what the opacity of hidden areas is for the DM.
+Players will always have this setting as 1.0 for themselves and cannot edit this.
 
-Do remember that in fake player mode you'll also be stuck on 1.0
+In fake player mode, you'll also be stuck on 1.0.
 
 ### Vision Mode
 
 _This is a technical option that I intend to remove in the next release of PlanarAlly._
 
-Historically PlanarAlly calculated vision with bounding volume hierarchies or bvh for short.
-Later on a different method was introduced based on triangulation. This was initially an optional experimantal mode you could select using this vision mode setting.
+Historically, PlanarAlly calculated vision with bounding volume hierarchies, or bvh for short.
+Later, a different method was introduced based on triangulation. 
+This was initially an optional, experimental mode you could select using this vision mode setting.
 
-As time went on however the triangle mode became the default as it is much more potent.  The option to choose the older mode is still there for legacy reasons, but is not being maintained currently.
+As time went on, however, the triangle mode became the default, as it is much more potent.  
+The option to choose the older mode remains for legacy reasons. 
+It’s not being maintained currently.
 
 ### Minimal/Maximal vision
 
-These settings are additional tuning factors to the vision system and determine limits to the vision of all tokens.
-
-Suppose there is one giant open field that stretches 5000 ft. and is completely illuminated. One could expect the entire area to be visible to your token standing in the middle of this field.  There are however limitations to the human (and to most non-human, for that matter) eye, hence the introduction of these settings.
+These settings are more tuning factors to the vision system. 
+They determine limits to the vision of all tokens.
 
+Suppose there is one giant, open field that stretches 5000 ft. and is completely illuminated.
+One could expect the entire area to be visible to your token standing in the middle of this field.
+There are, however, limitations to the human and nonhuman eye, hence the introduction of these settings.
 
-Everything up to the minimum vision range can be seen as usual. Everything past the maximum range can not be seen and is fog of war.  Everything in between is shown in a diminishing opacity as you go further towards the edge of vision.
+Everything up to the minimal vision range can be seen as usual. 
+Everything past the maximal range cannot be seen and is fog of war.  
+Everything in between is shown in a diminishing opacity as you go further towards the edge of vision.
```

Comment:
```suggestion
Everything in between is shown in an increasing opacity as you go further towards the edge of vision.
```

If I'm not mistaken (I may be, though), opacity is "not see-through-able" / the opposite of transparency, and, thus, towards the edge of the vision, opacity increases.

One could, of course, also show `in a diminishing transparency`.